README.md

```diff
@@ -235,41 +235,82 @@ $
 
 Multiple `raw` and `parsed` files are supported per directory, and are encouraged, as there are differences depending on version, length, etc. Additional test files and more real life data helps ensure backwards compatibility is maintained as each template is updated and merged into the repo.
 
+All YAML files must adhere to the YAML standards defined in the `.yamllint` file in the root directory. Yamllint provides thorough documentation of their configuration settings [here](https://yamllint.readthedocs.io/en/stable/rules.html). 
+
+##### DEVELOPMENT HELPER SCRIPTS
+
+We have provided a cli utility to assist with properly building the parsed files. All arguments that can be passed to the script are mutually exclusive (i.e. you can only pass one argument). The file can be made executable with the `chmod +x development_scripts.py` command. The arguments are:
+
+  * `-y`: Takes the path to a YAML file and ensures that the file adheres to the .yamllint settings
```

Comment:
I'm not sure `-y` works exactly as intended. I didn't leave a space between a `#` and the words, but it kept them without a space which appears to be one of the restraints within the `.yamllint` file. If I'm misunderstanding, please let me know.

e.g. `#test` instead of the proper formatting `# test`

Maybe update the verbiage as well. I thought it would spit out something saying that the file wasn't in adherence to the `yamllint` file. Along that note, some kind of output would be great as well. "Formatting completed/No formatting required", etc.

Reply:
I think we would probably need to see if ruamel supports comment spacing, or see about adding a custom representer to handle that use case.

I don't mind adding a print statement to say it is complete, but would say that should be assumed if the process exited without errors. The actual scripts are simple, and just rewrite the existing YAML file regardless of the correctness of the file.